crates/harness/src/monolith.rs

```diff
@@ -85,7 +85,10 @@ impl Monolith {
                 loop {
                     let (stream, _) = _listener.accept().await.unwrap();
                     let mut ws = tokio_tungstenite::accept_async(stream).await.unwrap();
-                    let init = M2BInit { port: http_port };
+                    let init = M2BInit {
+                        port: http_port,
+                        region: "unknown".into(),
```

Comment:
what does .into() do?

Reply:
It comes from the `Into` trait. https://doc.rust-lang.org/std/convert/trait.Into.html

You can read more in depth about `From`/`Into` here: https://practice.rs/type-conversions/from-into.html Specifically, this `into()` converts a `&'static str` into a `String`.